.github/workflows/frontend-docker-images.yml

```
@@ -0,0 +1,39 @@
+name: Build and Push Frontend Next.js Docker Images
+
+on:
+  schedule:
+    - cron: '0 3 * * 0' # Every Sunday at 3am UTC
+  workflow_dispatch:
+    # Allow manual trigger
+
+jobs:
+  # Job: build-frontend-base
+  # This job builds a docker image for the frontend-base repository and pushes the docker image to GitHub Packages.
+  build-frontend-base:
+    name: Build and Push frontend-base
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@e468171a9de216ec08956ac3ada2f0791b6bd435 # v3.11.1
```

Any reason not to do something like this instead?

```suggestion
        uses: docker/setup-buildx-action@v3.11.1
```

---

This is to mitigate a supply chain attack where the tagged version is replaced with with a malicious version on the same number.